pipeline/tests/pil.rs

```diff
@@ -54,19 +35,6 @@ fn test_invalid_witness_halo2mock() {
         .unwrap();
 }
 
-#[test]
-#[should_panic = "called `Result::unwrap()` on an `Err` value: [\"Proof is invalid\"]"]
-#[cfg(feature = "halo2")]
-fn test_invalid_witness_halo2() {
-    let f = "pil/trivial.pil";
-    Pipeline::default()
-        .from_file(resolve_test_file(f))
-        .set_witness(vec![("main.w".to_string(), vec![Bn254Field::from(0); 4])])
```

Comment:
is this not tested anymore?

Reply:
It is, `test_invalid_witness()` above calls `assert_proofs_fail_for_invalid_witnesses()` which tests Pilcom, eStark and Halo2 prover. The only thing it doesn't test is Halo2-Mock, which is why we keep the test above.